hack/secrets.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Please add `set -eu` so it will instantly fail when any commands fail or when there are undefined variables